autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -135,7 +135,8 @@ private DistroXAutoscaleClusterResponse updateClusterAutoScaleState(Cluster clus
 
     private DistroXAutoscaleClusterResponse updateClusterAutoScaleConfig(Cluster cluster,
             DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
-
+        alertValidator.validateScheduleWithStopStart(cluster, autoscaleClusterRequest);
```

Comment:
This is already invoked 5 lines down.

Reply:
I purposefully added it before this for doing the null value check and other checks before changing the value and also kept after it so that if in any case anything is left over than we can validate it again.

Reply:
Removed